### PR TITLE
Update to v0.19.0

### PR DESCRIPTION
**Pull request by sugh01**

### CODE REVIEW

File: charts/charon/Chart.yaml [hunk 1]
```diff
@@ -29,10 +29,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.8
+version: 0.3.0
```
Comment (LukeHackett12) [inline charts/charon/Chart.yaml:32]: whichever way we decide consistency is needed I think

**Review by LukeHackett12 — DISMISSED** [review]
lgtm % one comment

File: charts/charon/Chart.yaml (hunk 1 above)
Comment (sugh01) [reply charts/charon/Chart.yaml:32]: if i understood correctly, this is `charon` chart, so there may be a possibility of operators running incompatible charts and running into a conflict.
But this is not the case with `charon-cluster` chart, as it is a standalone chart (more like a CDVC)

Comment (sugh01) [reply charts/charon/Chart.yaml:32]: but for the benefit of doubt, can make the versions consistent.

**Review by LukeHackett12 — APPROVED** [review]

**Pull request merged**